lib/capistrano/data_migrate/migrate.rb

```diff
@@ -0,0 +1,42 @@
+Rake::Task["deploy:migrate"].clear
```

Comment:
will this override for non-capistrano tasks? maybe I don't know cap well enough, perhaps a little explanation in the readme would be helpful